From 8cd2eb5038c79ed17304c0f58c45c961456eb22b Mon Sep 17 00:00:00 2001
From: Ryohei Ueda <r.ueda@gitai.tech>
Date: Thu, 20 Feb 2025 11:42:59 -0800
Subject: [PATCH] Disable format-security flag

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 276329db..be30cc53 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -16,7 +16,7 @@ export LDFLAGS=
 export PKG_CONFIG_PATH=@(InstallationPrefix)/lib/pkgconfig
 # Explicitly enable -DNDEBUG, see:
 # 	https://github.com/ros-infrastructure/bloom/issues/327
-export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG
+export DEB_CXXFLAGS_MAINT_APPEND=-DNDEBUG -Wno-error=format-security
 ifneq ($(filter nocheck,$(DEB_BUILD_OPTIONS)),)
 	BUILD_TESTING_ARG=-DBUILD_TESTING=OFF -DCATKIN_ENABLE_TESTING=OFF
 endif
-- 
2.43.0

